navigation/screens/home.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import {
+  StyleSheet,
+  Image,
+  View,
+  Dimensions,
+  TouchableOpacity,
+} from "react-native";
+import { FontAwesome } from "@expo/vector-icons";
+import { useState, useEffect } from "react";
+import {
+  getAllBathrooms,
+  getBathroomProfile,
+} from "@/controllers/bathroomController";
+import { getUserProfile } from "@/controllers/userController";
+
+/* eslint-disable */
+const map = require("../../assets/images/map.jpg");
+
+export default function MainScreen() {
+  /*gets bathroom profile information*/
+  const [bathroomID, setBathroomID] = useState(0);
```

Comment:
this is not state. is it a property of the icon representing the bathroom with the corresponding ID